documents/src/pages/elements/chart.md

```diff
@@ -1483,7 +1483,7 @@ ChartJS.register(
 );
 ```
 
-EF provides center label plugin for doughnut chart. To use it:
+Another example is to import only required modules for creating doughnut chart.
```

Comment:
```suggestion
Another example is to using center label plugin for creating doughnut chart.
```

Reply:
The purpose of this example is showing how to use our plugin, so I want to keep 'center label plugin' message to notice.